src/main/java/org/opentripplanner/standalone/configure/ConstructApplication.java

```diff
@@ -180,8 +179,8 @@ private void setupTransitRoutingServer() {
     }
 
     if (OTPFeature.SandboxAPIGeocoder.isOn()) {
-      LOG.info("Creating debug client geocoder lucene index");
-      LuceneIndex.forServer(createServerContext());
+      LOG.info("Initializing geocoder");
+      this.factory.luceneIndex();
```

Comment:
The factory create a new instance every time you call it? I do not think this code has any effect?

Reply:
The LuceneIndex is annotated as `@Singleton`: https://github.com/opentripplanner/OpenTripPlanner/blob/a714a532e6ef0404e001fbd9519127a8a2935126/src/ext/java/org/opentripplanner/ext/geocoder/configure/GeocoderModule.java#L19

The idea here is to initialize the geocoder eagerly rather than on the first request.